src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -315,6 +314,11 @@ private int instances(AppDeploymentRequest request) {
 		if (route(request) != null) {
 			manifest.route(Route.builder().route(route(request)).build());
 		}
+		if(getDockerImage(request) != null){
```

Comment:
Was this change to make it more clear that they are mutually exclusive or did a test fail when trying to set both of them?